app/controllers/core_induction_programme_controller.rb

```diff
@@ -4,4 +4,17 @@ class CoreInductionProgrammeController < ApplicationController
   def show
     @course_years = CourseYear.all
   end
+
+  def download_export
+    unless @current_user&.admin?
+      redirect_to cip_path and return
```

Comment:
When you redirect you don't need the and return unlike when you render

Reply:
I think if you don't return, the method keeps going :/ At least failing tests think so.

I refactored it to not need a return though.